src/current/_includes/cockroachcloud/metric-names-serverless.md

```diff
@@ -0,0 +1,74 @@
+`jobs.create_stats.currently_running` | Number of create_stats jobs currently running
+
+{% comment %}
+### Distributed SQL (DistSQL)
+
+Name | Description
+-----|-----
+`distsender.batches.partial` | Number of partial batches processed after being divided on range boundaries
+`distsender.batches` | Number of batches processed
+`distsender.batch_requests.replica_addressed.bytes` | Total byte count of replica-addressed batch requests processed
+`distsender.batch_responses.replica_addressed.bytes` | Total byte count of replica-addressed batch responses received
+`distsender.batch_requests.cross_region.bytes` | Total byte count of replica-addressed batch requests processed cross region when region tiers are configured
+`distsender.batch_responses.cross_region.bytes` | Total byte count of replica-addressed batch responses received cross region when region tiers are configured
+`distsender.batch_requests.cross_zone.bytes` | Total byte count of replica-addressed batch requests processed cross zone within the same region when region and zone tiers are configured. However, if the region tiers are not configured, this count may also include batch data sent between different regions. Ensuring consistent configuration of region and zone tiers across nodes helps to accurately monitor the data transmitted.
+`distsender.batch_responses.cross_zone.bytes` | Total byte count of replica-addressed batch responses received cross zone within the same region when region and zone tiers are configured.	However, if the region tiers are not configured, this count may also include batch data received between different regions. Ensuring consistent	configuration of region and zone tiers across nodes helps to accurately monitor the data transmitted.
+`distsender.batches.async.sent` | Number of partial batches sent asynchronously
+`distsender.batches.async.throttled` | Number of partial batches not sent asynchronously due to throttling
+`distsender.rpc.sent.nextreplicaerror` | Number of replica-addressed RPCs sent due to per-replica errors
+`distsender.errors.notleaseholder` | Number of NotLeaseHolderErrors encountered from replica-addressed RPCs
+`distsender.errors.inleasetransferbackoffs` | Number of times backed off due to NotLeaseHolderErrors during lease transfer
+`distsender.rangefeed.total_ranges` | Number of ranges executing rangefeed. This counts the number of ranges with an active rangefeed.
+`distsender.rangefeed.catchup_ranges` | Number of ranges in catchup mode. This counts the number of ranges with an active rangefeed that are performing catchup scan.
+`distsender.rangefeed.error_catchup_ranges` | Number of ranges in catchup mode which experienced an error
+`distsender.rangefeed.restart_ranges` | Number of ranges that were restarted due to transient errors
+`distsender.rangefeed.restart_stuck` |Number of times a rangefeed was restarted due to not receiving timely updates (kv.rangefeed.range_stuck_threshold cluster setting)
+
+### Resource Units (RUs)
```

Comment:
We removed these metrics for V1. @THardy98 FYI

Reply:
Deleted commented lines.